pkg/pluginmanager/manager.go

```diff
@@ -767,13 +761,17 @@ func installOrUpgradePlugin(p *discovery.Discovered, version string, installTest
 		}()
 
 		// Initialize the spinner
-		sow, spinnerErr = component.NewOutputWriterSpinnerWithSpinnerOptions(os.Stderr, component.TableOutputType, installingMsg, true, spinnerOptions)
+		spinner, spinnerErr = component.NewSpinner(component.WithStdout(os.Stderr),
+			component.WithOutputFormat(component.TableOutputType),
+			component.WithSpinnerText(installingMsg),
+			component.WithSpinnerStarted(),
+			component.WithSpinnerFinalText(installedMsg, log.LogTypeINFO))
 		if spinnerErr != nil {
 			log.V(6).Infof("Unable to initialize spinner: %v", spinnerErr.Error())
 			log.Info(installingMsg)
 		}
-		if sow != nil {
-			defer sow.StopSpinner()
+		if spinner != nil {
+			defer spinner.StopSpinner()
 		}
 		// Start a goroutine that listens for interrupt signals
 		go func(s component.OutputWriterSpinner) {
```

Comment:
Can you change this type to the new `Spinner`?

Reply:
As we have renamed `Spinner` back to `OutputWriterSpinner` in plugin-runtimr this change is not needed anymore.